### ISSUE TITLE
[RIP-46] Add logging metrics exporter

### ISSUE THREAD

**Issue by ShadowySpirits** [apache/rocketmq#5678]
For now, we need an Opentelemetry collector or Prometheus to explore metrics. It's useful to add [logging exporter](https://github.com/open-telemetry/opentelemetry-java/tree/main/exporters/logging) for debugging purposes.

**Comment by Bughue** [comment]
I would like to give it a try, can you assign this issue to me

**Comment by Bughue** [comment]
> For now, we need an Opentelemetry collector or Prometheus to explore metrics. It's useful to add [logging exporter](https://github.com/open-telemetry/opentelemetry-java/tree/main/exporters/logging) for debugging purposes.

I found that there is already a rocketmq-exporter(https://github.com/apache/rocketmq-exporter), what's the difference?

**Comment by ShadowySpirits** [comment]
> > For now, we need an Opentelemetry collector or Prometheus to explore metrics. It's useful to add [logging exporter](https://github.com/open-telemetry/opentelemetry-java/tree/main/exporters/logging) for debugging purposes.
> 
> I found that there is already a rocketmq-exporter(https://github.com/apache/rocketmq-exporter), what's the difference? Do they have different purposes?

This wiki [Observability Improvement for RocketMQ](https://github.com/apache/rocketmq/wiki/RIP-46-Observability-improvement-for-RocketMQ) can answer your questions.

**Comment by ShadowySpirits** [comment]
No, this issue aims to add a log exporter. Currently, we have otlp exporter and Prometheus exporter.

https://github.com/apache/rocketmq/blob/5744721638a2b963cb2dc2fe83bdbef8b22d52e8/common/src/main/java/org/apache/rocketmq/common/BrokerConfig.java#L346-L349

**Comment by Bughue** [comment]
> No, this issue aims to add a log exporter. Currently, we have otlp exporter and Prometheus exporter.
> 
> https://github.com/apache/rocketmq/blob/5744721638a2b963cb2dc2fe83bdbef8b22d52e8/common/src/main/java/org/apache/rocketmq/common/BrokerConfig.java#L346-L349

ok~thx

**Comment by ShadowySpirits** [comment]
Closed as completed via #6234
